src/protocol_upgrade_checker.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import time
+
+from web3 import Web3
+
+from src import variables
+from src.contracts import contracts
+from src.metrics.healthcheck_server import pulse
+
+logger = logging.getLogger(__name__)
+
+
+def wait_for_withdrawals(w3: Web3):
```

Comment:
We need to implement the contract and the consensus versions check anyway, after which this method will be unnecessary.